src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm._Sve
+{
+    public static partial class Program
```

Comment:
Please give this a name consistent with other regression tests, i.e. `Runtime_<Id>`. You can remove the namespace too

Reply:
You can also remove `partial`

Reply:
Done.